packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1149,6 +1149,7 @@ abstract class RangeSliderThumbShape {
     SliderThemeData sliderTheme,
     Thumb thumb,
     bool isPressed,
+    RangeValues values
```

Comment:
Include a trailing comma here. 

This is a breaking change for RangeSliderThumbShape subclasses. @clocksmith is also looking extending this API; probably best to do so all at once.

CC @clocksmith @JoseAlba